app/controllers/confirmations_controller.rb

```diff
@@ -31,9 +31,9 @@ def new
     @email = current_user&.unconfirmed_email || current_user&.email
   end
 
-  def after_confirmation_path_for(resource_name, resource)
+  def after_confirmation_path_for(resource_name, _resource)
```

Comment:
why keep the `_resource` as optional?
Is that so we don't have to go back and change everywhere `after_confirmation_path_for` is used?
Or is that because devise will stick the resource in somewhere irrespective of what we call in our code?

Reply:
Yeah, this method is called by devise